### ISSUE TITLE
replace multi lines

### ISSUE THREAD

**Issue by kat75** [notepad-plus-plus/notepad-plus-plus#3693]
### Description of the Issue

Actually, it's possible to search multilines but not to replace them with multilines and or empty lines in windows format, in a file or multifiles in directory
it will be great to have this possibility
actually its just possible in unix format

### Steps to Reproduce the Issue

search
123

456
replace
789

1011

### Expected Behavior

789

1011

### Actual Behavior

789

789

1011

### Debug Information

this work only in unix format but not in windows format, it will be great that it works in windows format to replace in a file or files in a diectory
![yniuryv](https://user-images.githubusercontent.com/28734790/30001102-c7c89df2-9083-11e7-9649-eba11366c38e.gif)

**Comment by azlux** [comment]
It's because Linux format use `\n` for new line, and windows use `\r\n` . You can replace new line with the extended search.

**Comment by kat75** [comment]
yes thanks azlux, but seems not possible to paste multilines in the replace box juste possible to put on 1 line like this

the text which\r\nreplaces the one above

**Comment by CU61** [comment]
But it worked many years ago with earlier versions of NPP. When did that change?
Then I would rather go back to an outdated version where multi-lines work.
Thank you.
